SeaPublicWebsite/SeaPublicWebsite.csproj

```diff
@@ -31,4 +31,8 @@
         </None>
     </ItemGroup>
 
+    <ItemGroup>

```

Comment:
I don't think you want this

Reply:
Huh, I wonder why it got added here... yeah, it's not needed I'll remove it.